workspace/workspace.go

```diff
@@ -102,3 +103,53 @@ func (ws Workspace) Locate(exercise string) ([]string, error) {
 	}
 	return paths, nil
 }
+
+func (ws Workspace) SolutionPath(exercise, solutionID string) (string, error) {
+	paths, err := ws.Locate(exercise)
+	if !IsNotExist(err) && err != nil {
+		return "", err
+	}
+
+	return ws.DetermineSolutionPath(paths, exercise, solutionID, PathContainsSolution)
+}
+
+// TODO: PathContainsSolution might need a different name.
```

Comment:
I would love some help thinking about the naming (this will need to be updated even if the names stay the same, as this is not passing golint at the moment).